### PR TITLE
Add github pull requests support

### PR DESCRIPTION
**Pull request by stefk**
Install hub inside the weblate container and expose `GITHUB_USERNAME` through an environment variable.

### CODE REVIEW

File: weblate/Dockerfile [hunk 1]
```diff
@@ -23,6 +23,9 @@ RUN env DEBIAN_FRONTEND=noninteractive apt-get update \
   && apt-get clean
 
 # Examples
+RUN curl -L https://github.com/github/hub/releases/download/v2.2.9/hub-linux-amd64-2.2.9.tgz | tar xzv && \
+  cp hub-linux-amd64-2.2.9/bin/hub /usr/bin
+
```
Comment (nijel) [inline weblate/Dockerfile]: This way you leave all the files in the docker image, what is IMHO not a good idea. Doing something like `tar xzv --wildcards hub-linux*/bin/hub` should do better job. Also you should remove the hub-linux-amd64-2.2.9 directory after copying the file out.

Comment (stefk) [inline weblate/Dockerfile]: @nijel goods points, updated.

**Comment by nijel** [comment]
Merged, thanks!

Can you please also add example how to set it up with docker? I've added basic docs in https://github.com/WeblateOrg/weblate/commit/1eb3c92edbbbd5396a86b6f2008bd5aff6f97996, but having exact command to invoke for the Docker in https://docs.weblate.org/en/latest/vcs.html#setting-up-hub would be useful.

**Comment by stefk** [comment]
@nijel done in https://github.com/WeblateOrg/weblate/pull/1359 but I'm not sure if that's exactly what you wanted (I'm also not familiar with rst format).

**Pull request merged**